TC/TC_01_settings.req

```diff
@@ -0,0 +1,11 @@
+file "motor_settings.req" P=TE:NDLT1542:, M=MOT:MTR0101
+file "motor_settings.req" P=TE:NDLT1542:, M=MOT:MTR0102
+file "motor_settings.req" P=TE:NDLT1542:, M=MOT:MTR0103
+file "motor_settings.req" P=TE:NDLT1542:, M=MOT:MTR0104
+file "motor_settings.req" P=TE:NDLT1542:, M=MOT:MTR0105
+file "motor_settings.req" P=TE:NDLT1542:, M=MOT:MTR0106
+file "motor_settings.req" P=TE:NDLT1542:, M=MOT:MTR0107
+file "motor_settings.req" P=TE:NDLT1542:, M=MOT:MTR0108
+file "motor_settings.req" P=TE:NDLT1542:, M=MOT:MTR0109
+file "motor_settings.req" P=TE:NDLT1542:, M=MOT:MTR0110
+file "motor_settings.req" P=TE:NDLT1542:, M=MOT:MTR0111
```

Comment:
I wonder is we can simplify this using https://epics-modules.github.io/autosave/autoSaveRestore.html#autosavebuild-automatic-request-file-generation and a `single_axis_tc.req` file - at the moment this is another hard-coded axis limit

Reply:
i think that .req file is generated by line 21 and 43 of the lua - perhaps for now i could just unstage that file and add it to the gitignore